.evergreen.yml

```diff
@@ -150,7 +150,11 @@ functions:
       working_dir: spruce
       script: |
         export PATH=/opt/nodejs/node-v16.13.0-linux-x64/bin:$PATH
-        ./node_modules/.bin/cypress run --record --key ${cypress_record_key} --reporter junit
+          cd /tmp
+          curl -L -o 'my_file.zip' 'https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Linux_x64%2F1002917%2Fchrome-linux.zip?generation=1652398617890755&alt=media'
+          unzip my_file.zip
+          cd -
+        ./node_modules/.bin/cypress run --browser /tmp/chrome-linux/chrome --record --key ${cypress_record_key} --reporter junit
```

Comment:
I wonder if we can just run `yarn cy:run` instead of accessing the executable directly